### PR TITLE
ADAPT-4436 | @rebeccahongsf | create api status page for gateway auth

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# NOT READY FOR REVIEW

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to [`/api/auth/status`](https://deploy-preview-316--stanford-alumni.netlify.app/api/auth/status)

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

# Associated Issues and/or People
- ADAPT-4436

### CODE REVIEW

**Comment by sherakama** [comment]
@rebeccahongsf please move this work to https://github.com/SU-SWS/adapt-sso-sp
That way we can stay consistent with where we are monitoring things. 

Also, you won't need SAML auth and can just use a token to protect the endpoint

As a design thing, I think we are going to have to monitor each endpoint independently so we will likely need dozens of new URLs.

**Comment by rebeccahongsf** [comment]
> @rebeccahongsf please move this work to https://github.com/SU-SWS/adapt-sso-sp That way we can stay consistent with where we are monitoring things.
> 
> Also, you won't need SAML auth and can just use a token to protect the endpoint
> 
> As a design thing, I think we are going to have to monitor each endpoint independently so we will likely need dozens of new URLs.

Ah, good to know! 😄  I'll close this out and move the work over. Thanks @sherakama!
